content/birds/olu.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Olu Niyi-Awosusi
+sub: Javascript Developer
+bird: olu
+date: 2021-06-07
+# image:
+#   src: [`birds/bird.jpg`]
```

Comment:
@oluoluoxenfree if you would prefer to not have a header image, then we probably will want to remove this commented out code. We can add a hero image at any time in the future if you have one you want to use down the road.

Reply:
(I did the commenting-out, just to keep this from throwing an error. I think we want to have a header image if we can)

Reply:
@mirisuzanne do i need to delete this or add a header? or was it just a heads up?

Reply:
@oluoluoxenfree I think ideally we would add a header image here, like on the other bird pages. Do you have a photo we could use for that?

Reply:
@mirisuzanne nothing immediately comes to mind 😞 is it okay to just leave it blank?

Reply:
@oluoluoxenfree I think we can make it work. I tried adding your sketch as a header image, just so it's a bit more similar to the other bird pages. Does that look ok to you?